src/wallet/rpcwallet.cpp

```diff
@@ -2834,7 +2834,7 @@ UniValue bumpfee(const JSONRPCRequest& request)
 
     // If the output would become dust, discard it (converting the dust to fee)
     poutput->nValue -= nDelta;
-    if (poutput->nValue <= poutput->GetDustThreshold(::minRelayTxFee)) {
+    if (poutput->nValue <= poutput->GetDustThreshold(dustRelayFee)) {
```

Comment:
yes, that has annoyed me a few times as well and I was going to change it, but @jtimon has #9279 which I was hoping would eventually get merged and make it go away all together.